src/store/chat/index.ts

```diff
@@ -12,9 +12,13 @@ const initialState: ChatState = {
 
 export enum SagaActionTypes {
   CloseConversationErrorDialog = 'chat/saga/closeConversationErrorDialog',
+  ValidateAndSetActiveConversationId = 'channelsList/saga/ValidateAndSetActiveConversationId',
 }
 
 const closeConversationErrorDialog = createAction(SagaActionTypes.CloseConversationErrorDialog);
+export const validateAndSetActiveConversationId = createAction<{ id: string }>(
```

Comment:
This feels like an awkward name. The caller shouldn't really know what's involved in specifying the active conversation. But I'm struggling to come up with something that's not already too overloaded. `setActiveConversationId` is already how we do the actual data set.... `activateConversation`, perhaps?.... :thinking:

Reply:
Maybe the redux setter could change to `rawSetActiveConversationId`? Does that make that one clearly different and then we can use `setActiveConversationId` here?... just throwing out ideas.

Reply:
`rawSetActiveConversationId` & `setActiveConversationId` makes more sense